etcd-client/async_etcd_client.py

```diff
@@ -0,0 +1,71 @@
+import aio_etcd as etcd
+import asyncio
+import sys
+
+
+class AsyncEtcdClient:
+    def __init__(self, host, port):
+        self.host = host
+        self.port = port
+
+    async def get(self, key, prefix=False):
+        client = etcd.Client(self.host, self.port)
+        try:
+            if prefix:
+                resp = await client.read(key, recursive=True)
+                for child in resp.children:
+                    print("%s: %s" % (child.key, child.value))
+                return
+            resp = await client.read(key)
+            print("%s: %s" % (key, resp.value))
+        except etcd.EtcdKeyNotFound:
+            print('Key "%s" not found' % key)
+
+    async def set(self, key, value, swap=False, oldvalue=""):
+        client = etcd.Client(self.host, self.port)
+        try:
+            if swap:
+                print("Swapping {} with {}".format(oldvalue, value))
+                await client.write(key, value, prevValue=oldvalue)
+                return
+            await client.write(key, value, prevExist=False)
+        except etcd.EtcdKeyNotFound:
+            print('Key "%s" not found' % key)
```

Comment:
It's useful to print error messages to sys.stderr instead of the default stdout: (`print("message", file=sys.stderr)`)